metrics/metrics.go

```diff
@@ -27,7 +27,7 @@ limitations under the License.
 //
 // last_checked_timestamp_lag_minutes - shows how slow we get messages from clusters
 //
-// produced_messages - total number of produced messages
+// produced_messages - total number of produced messages sent to Payload Tracker's Kafka topic
```

Comment:
I thought it's not about payload tracker..

Reply:
As far as I can tell, it is. The only place where we are increasing the value of this counter is un `producer.produceMessage` method (OBVIOUS :-D), and the sent message is a `PayloadTrackerMessage`

Reply:
Yup, that's the reason why I was scared by this metric on Grafana :)